HTAN.model.csv

```diff
@@ -775,4 +775,25 @@ Phospho Site,The protein site for a phosphoprotein targeting antibody. Report AA
+PMID,The PubMed identifier associated with the publication (applicable to published manuscripts),,,,FALSE,Publication,,,
+Data Type,"Types of data associated with the content. Fill out Other Data Type Specified, if not on the list.","10X Visium, Bulk DNA, Bulk RNA-seq, CyCIF, H&E, IMC, LC-MS-MS, LC-MS3, MIBI, MxIF, Shotgun MS (lipidomics), mIHC, scATAC-seq, scRNA-seq, t-CyCIF, Demographics",,,TRUE,Publication,,,list
+Other Data Type Specified,Other types of data associated with the content.,,,,FALSE,Publication,,,list
+Supporting Link,Relevant external links associated with the content (e.g external datasets used for validation). Please note: Supporting Links and Supporting Link Descriptions are provided by authors and are not verified by the NIH NCI or the HTAN DCC. This information and any linked data should only be shared by an authorized individual(s) in accordance with the terms of the HTAN data sharing agreements and policies and/or any other applicable agreement(s).,,,,FALSE,Publication,,,
+Supporting Link Description,Description of relevant external links associated with the publication (e.g An external mouse dataset used for validation). Please note: Supporting Links and Supporting Link Descriptions are provided by authors and are not verified by the NIH NCI or the HTAN DCC. This information and any linked data should only be shared by an authorized individual(s) in accordance with the terms of the HTAN data sharing agreements and policies and-or any other applicable agreement(s).,,,,FALSE,Publication,,,
+Tool,Were any software or computational tools generated for this content,"Yes, No",,,TRUE,Publication,,,
```

Comment:
Do we want them to list the tools if they did?

Reply:
@aclayton555 appreciate your thoughts on this one

Reply:
I don't think we need to have them list the tools. At this stage, a response of "Yes" is really just a prompt for us to follow-up with them to complete the tool curation form. The tool curation form collects the PMID for any relevant publications associated with a tool, so that linkage will be maintained (if a publication exists). We may revisit this, however, in the context of the longer term plan for collecting both publication and tool information.

Reply:
Thanks! I'll resolve this comment